prody2/protocols/protocol_atoms.py

```diff
@@ -39,7 +39,7 @@
 
 from pyworkflow.utils import *
 from pyworkflow.protocol.params import (PointerParam, StringParam, FloatParam,
-                                        EnumParam, LEVEL_ADVANCED)
+                                        BooleanParam, EnumParam, LEVEL_ADVANCED)
 
 import prody
 from prody import LOGGER
```

Comment:
Should we go for the standard way of logging

Reply:
Yes. We have some in #34 and I'll do the same for the new things here.